docroot/.gitignore

```diff
@@ -4,3 +4,4 @@ sites/*/settings*.php
 # Ignore paths that contain user-generated content.
 sites/*/files
 sites/*/private
+*.sass-cache
```

Comment:
I meant that we don't need add files/folders, which names starts from a dot, to .gitignore every time.